### PR TITLE
Felix/seed data creation

### PR DESCRIPTION
**Pull request by Felix-Cadiz**
Formatted index.db and added seed data for wines in init_db. Cheeses functions will be added later.

### CODE REVIEW

**Review by kbonifacio — APPROVED** [review]
lgtm

**Review by Felix-Cadiz — COMMENTED** [review]
LGTM! Thanks for the great descriptions!

File: db/init_db.js [hunk 1]
```diff
@@ -447,6 +447,78 @@ async function populateInitialData() {
         price: "$9",
         category: "cheese"
       },
+      {
+        productId: 37,
+        name: "Extra Mature Real Yorkshire Wensleydalee",
```
Comment (kbonifacio) [inline db/init_db.js:402]: Please change Wensleydalee to **Wensleydale** unless this was intentional then ignore.

**Review by kbonifacio — CHANGES_REQUESTED** [review]
Just caught a spelling error for the Wensleydale cheese in the name. Other than that the code looks good for the new cheeses.

**Comment by Felix-Cadiz** [comment]
Nice catch and even better descriptions! Everything else looks good, especially id:44.

File: db/init_db.js (hunk 1 above)
Comment (kbonifacio) [reply db/init_db.js:402]: Pushing through for code review, will resolve later tonight.

**Comment by kbonifacio** [comment]
Will resolve cheese update requests later tonight during the breakout. Pushing through for code review.

**Pull request merged**